std_msgs/String robot_name
geometry_msgs/Pose2D location
